HetDepot/Controllers/Visitor/RequestAuthenticationController.cs

```diff
@@ -18,13 +18,13 @@ public RequestAuthenticationController(Tour tour, bool forGroup = false) : base(
 		public override void Execute()
 		{
 			var title = Program.SettingService.GetConsoleText("consoleVisitorRequestCodeSelectedTour", new() {
-				["time"] = _tour.StartTime.ToString(),
+				["time"] = _tour.GetTime(),
 			});
 			var textToUser = Program.SettingService.GetConsoleText("consoleLogonOpeningWelcome");
 
 			if (_forGroup)
 			{
-				title = Program.SettingService.GetConsoleText("consoleVisitorRequestCodeSelectedTourForGroup") + _tour.StartTime;
+				title = Program.SettingService.GetConsoleText("consoleVisitorRequestCodeSelectedTourForGroup") + _tour.GetTime();
```

Comment:
Kan die time beter een component {time} worden in plaats van altijd erachter?

Zoiets:

```
title = Program.SettingService.GetConsoleText("consoleVisitorRequestCodeSelectedTourForGroup", new()
{
    ["time"] = _tour.StartTime,
});
```